dockerfiles/README.md

```diff
@@ -75,41 +78,25 @@ The relationship for the targets is as follows:
            |                 |
            |                 |
            v                 |
-     +------------+          | 
-     | build-test |          |
+     +------------+          |
+     | all-tests  |          |
      +------------+          v
-                        +---------+  
+                        +---------+
                         |  build  |
                         +---------+
-
-
-The Dockerfile [Dockerfile.test](Dockerfile.test) is called by the make target
-`build-test` to test the image.
+```
 
 The target `test-container-image` builds the image and runs structure tests to test
-the images structure: 
+the images structure:
 
 ```Makefile
 test-container-image: build ## Target to test the final image
-	@$(CONTAINER_ENGINE) run --rm \
-		-v /var/run/docker.sock:/var/run/docker.sock \
-		-v $(CURDIR):/work \
-		 $(CTR_STRUCTURE_IMG) test \
-		--config /work/dockerfiles/structure-test.yaml \
-		-i $(IMAGE_NAME):$(IMAGE_TAG)
-```
-
-`test-app` builds the image for testing with tox:
-
-```Makefile
-test-app: build-test ## Target to test app with tox on docker
-	@$(CONTAINER_ENGINE) run --rm $(IMAGE_TEST)
+ @$(CONTAINER_ENGINE) run --rm \
+  -v /var/run/docker.sock:/var/run/docker.sock \
+  -v $(CURDIR):/work \
+   $(CTR_STRUCTURE_IMG) test \
+  --config /work/dockerfiles/structure-test.yaml \
+  -i $(IMAGE_NAME):$(IMAGE_TAG)
```

Comment:
Since you're modifying this, I guess it's better to write the real thing

```
test-container-image: build ## Target to test the final image
	@CONTAINER_ENGINE=$(CONTAINER_ENGINE) \
	CTR_STRUCTURE_IMG=$(CTR_STRUCTURE_IMG) \
	CURDIR=$(CURDIR) \
	IMAGE_NAME=$(IMAGE_NAME) \
	IMAGE_TAG=$(IMAGE_TAG) \
	$(CURDIR)/run-test-container-image.sh
```

Reply:
Removed in 2de8cc8